doc/compiling/linux.md

```diff
@@ -13,6 +13,7 @@
 | LuaJIT     | 2.0+    | Bundled Lua 5.1 is used if not present |
 | GMP        | 5.0.0+  | Bundled mini-GMP is used if not present |
 | JsonCPP    | 1.0.0+  | Bundled JsonCPP is used if not present |
+| Curl       | 7.68.0+ | Optional   |
```

Comment:
This version is from Jan 8, 2020.
This will mean Minetest cannot be built anymore on Debian 10, Ubuntu 18.04 and RHEL 8 derivatives.

Seems a bit too early to me, no?